js/scenes/Profile/ProfileContainer.js

```diff
@@ -0,0 +1,14 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
```

Comment:
You've imported connect into your new components but you're not using it. 
Set up ESLint to warn about unused imports. If you intend to use connect, you can connect the components.
Either remove the unused imports from all of your files or use them to connect the components.